docs/source/records_mover.rst

```diff
@@ -0,0 +1,26 @@
+records\_mover package
+======================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   records_mover.airflow
+   records_mover.records
+
+Module contents
+---------------
+
+.. automodule:: records_mover
+   :members:
+   :undoc-members:
+   :show-inheritance:
+   :exclude-members: Session
+
+   .. autoclass:: records_mover.Session
+      :members: records, set_stream_logging, get_db_engine, get_default_db_engine
```

Comment:
I documented only specific methods here.  In particular I didn't yet document the Creds hierarchy - it seemed like a whole different can of worms to lock down at this point.  As a result, the methods needed to do google sheets work aren't documented in Sphinx yet - I dropped a post-1.0 backlog item to deal with that.